code/modules/luck/luck_test.dm

```diff
@@ -0,0 +1,187 @@
+	yield = 6
+
+/*
+	lifespan = 60
+	maturation = 6
+	production = 6
+	yield = 6
+	potency = 5
+	ideal_light = 8
+	large = 0
+*/
+
+/datum/seed/clover/zeroleaf
+	name = "clover0"
+	products = list(/obj/item/weapon/reagent_containers/food/snacks/grown/clover/zeroleaf)
+	mutants = list("clover1")
+	plant_icon_state = "clover0"
+	seed_name = "zero-leaf clover"
+	display_name = "zero-leaf clover"
+
+/datum/seed/clover/oneleaf
+	name = "clover1"
+	products = list(/obj/item/weapon/reagent_containers/food/snacks/grown/clover/oneleaf)
+	mutants = list("clover0","clover3")
+	plant_icon_state = "clover1"
+	seed_name = "one-leaf clover"
+	display_name = "one-leaf clover"
+
+/datum/seed/clover/twoleaf
+	name = "clover2"
+	products = list(/obj/item/weapon/reagent_containers/food/snacks/grown/clover/twoleaf)
+	mutants = list("clover1","clover")
+	plant_icon_state = "clover2"
+	seed_name = "two-leaf clover"
+	display_name = "two-leaf clover"
+
+/datum/seed/clover/fourleaf
+	name = "clover4"
+	products = list(/obj/item/weapon/reagent_containers/food/snacks/grown/clover/fourleaf)
+	mutants = list("clover","clover5")
+	plant_icon_state = "clover4"
+	seed_name = "four-leaf clover"
+	display_name = "four-leaf clover"
+
+/datum/seed/clover/fiveleaf
+	name = "clover5"
+	products = list(/obj/item/weapon/reagent_containers/food/snacks/grown/clover/fiveleaf)
+	mutants = list("clover4","clover6")
+	plant_icon_state = "clover5"
+	seed_name = "five-leaf clover"
+	display_name = "five-leaf clover"
+
+/datum/seed/clover/sixleaf
+	name = "clover6"
+	products = list(/obj/item/weapon/reagent_containers/food/snacks/grown/clover/sixleaf)
+	mutants = list("clover5","clover6")
+	plant_icon_state = "clover6"
+	seed_name = "six-leaf clover"
+	display_name = "six-leaf clover"
+
+/datum/seed/clover/sevenleaf
+	name = "clover7"
+	products = list(/obj/item/weapon/reagent_containers/food/snacks/grown/clover/sevenleaf)
+	mutants = list("clover6")
+	plant_icon_state = "clover7"
+	seed_name = "seven-leaf clover"
+	display_name = "seven-leaf clover"
+
+/obj/item/seeds/cloverseed
+	name = "packet of clover seeds"
+	seed_type = "clover"
+	vending_cat = "weeds"
+
+/obj/item/seeds/cloverseed/zeroleaf
+	name = "packet of zero-leaf clover seeds"
+	seed_type = "clover0"
+
+/obj/item/seeds/cloverseed/oneleaf
+	name = "packet of one-leaf clover seeds"
+	seed_type = "clover1"
+
+/obj/item/seeds/cloverseed/twoleaf
+	name = "packet of two-leaf clover seeds"
+	seed_type = "clover2"
+
+/obj/item/seeds/cloverseed/fourleaf
+	name = "packet of four-leaf clover seeds"
+	seed_type = "clover4"
+
+/obj/item/seeds/cloverseed/fiveleaf
+	name = "packet of five-leaf clover seeds"
+	seed_type = "clover5"
+
+/obj/item/seeds/cloverseed/sixleaf
+	name = "packet of six-leaf clover seeds"
+	seed_type = "clover6"
+
+/obj/item/seeds/cloverseed/sevenleaf
+	name = "packet of seven-leaf clover seeds"
+	seed_type = "clover7"
```

Comment:
I'd move this entire chunk of code to code/modules/hydroponics/seeds.dm.
And perhaps rework the mutations a bit so you don't have to mess with 7 layers of mutations. You could cut on half the mutation work if you add more than one clover species to the mutations list, so you could mutate a 3 leaf clover into a 4 leaf normally or a 5 one if you get lucky with the mutation rolls.

Reply:
Will do. I'm keeping it there for convenience while I figure out how botany works.
Is there indeed a way to have weighted mutation rolls? Any examples of this with other plants?

Reply:
You can probably implement it really easily using simple prob(20)s or something, but if you add 2 or more species to the mutations list they each have an equal chance to be selected as the mutation. Trust me when I tell you that even without a weighted mutation roll it is quite hard to get lucky (heh), I've sometimes spent 10m trying to get novaflowers but 20 or so plants after all I have is unmutable teen sunflowers and moonflowers.

Reply:
Gotcha. I'll see if I can work it out.